docs/processes/coding standard/Wordpress/WPCS.md

```diff
@@ -0,0 +1,178 @@
+---
+position: 1
+title: WordPress Coding Standards
+---
```

Comment:
please add the sidebar title and tags it will be help full for navigating and searching